### ISSUE TITLE
Upload to USA doesn't work

### ISSUE THREAD

**Issue by nunyabiz1982** [gbrodman/order-tracking#28]
Using latest code and upload to USA still doesn't work even with the retry logic:

Found 68 total tracking numbers
Uploading tracking numbers...

DevTools listening on ws://127.0.0.1:61109/devtools/browser/dbd2038d-5bf4-4097-935e-a4fe96d7e1d1

DevTools listening on ws://127.0.0.1:61134/devtools/browser/bae0da43-87c0-4603-8261-fa83f4c10d59
[0812/233748.700:INFO:CONSOLE(1)] "500 - Internal Server Error.
undefined", source: https://usabuying.group/_next/6dd19a2c-4b58-4760-986b-5d11f998a8ac/main.js (1)
Received exception when uploading: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//*[contains(text(), ' Add')]"}
  (Session info: headless chrome=76.0.3809.100)

DevTools listening on ws://127.0.0.1:61162/devtools/browser/5241b203-688e-4d9f-ae97-e1261cc000ae
[0812/233820.919:INFO:CONSOLE(1)] "500 - Internal Server Error.
undefined", source: https://usabuying.group/_next/6dd19a2c-4b58-4760-986b-5d11f998a8ac/main.js (1)
Received exception when uploading: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//*[contains(text(), ' Add')]"}
  (Session info: headless chrome=76.0.3809.100)

DevTools listening on ws://127.0.0.1:61204/devtools/browser/400a3668-9634-45b2-87d7-8b3ece95f533
[0812/233853.308:INFO:CONSOLE(1)] "500 - Internal Server Error.
undefined", source: https://usabuying.group/_next/6dd19a2c-4b58-4760-986b-5d11f998a8ac/main.js (1)
Received exception when uploading: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//*[contains(text(), ' Add')]"}
  (Session info: headless chrome=76.0.3809.100)

DevTools listening on ws://127.0.0.1:61245/devtools/browser/5ee42709-0401-4cc7-9cac-b96574c64f94
[0812/233925.633:INFO:CONSOLE(1)] "500 - Internal Server Error.
undefined", source: https://usabuying.group/_next/6dd19a2c-4b58-4760-986b-5d11f998a8ac/main.js (1)

**Comment by gbrodman** [comment]
Do you receive that line about a "500 internal server error" for non-USA groups? It seems like that might be some sort of error on their end.

**Comment by nunyabiz1982** [comment]
I don't have any indication that it has even tried other groups.  These
trackings may have already been uploaded and there is nothing printed about
them

**Comment by gbrodman** [comment]
Right, I'm wondering if that log line about an internal server error is why USA is failing.

**Comment by gbrodman** [comment]
Changed the name since this was an auth failure with USA
